privacyidea/api/lib/postpolicy.py

```diff
@@ -710,7 +710,7 @@ def autoassign(request, response):
 
                                 g.audit_object.log(
                                     {"success": True,
-                                     "action_info":
+                                     "info":
```

Comment:
I changed this to ``info`` because the ``action_info`` value never ends up in the audit log.

Reply:
I think the texts you changed are ok. So you could directly add them to the "message" key.

       "message": u"user has no token, accepted by '{!s}'".format(policy_name)

I think the double entries of message and info are a bit too confusing.